src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -11,13 +11,32 @@
 
     class SymfonyProfiler {
         constructor() {
+            this.#reorderMainMenuItems();
             this.#createTabs();
             this.#createTableSearchFields();
             this.#createToggles();
             this.#createCopyToClipboard();
             this.#convertDateTimesToUserTimezone();
         }
 
+        #reorderMainMenuItems() {
+            /* reorder the main menu items to always display first the non-disabled items */
+            const mainMenuElement = document.querySelector('#menu-profiler');
+            const mainMenuItems = mainMenuElement.querySelectorAll('li');
+            const firstDisabledMenuItem = mainMenuElement.querySelector('li a > span.disabled')?.parentNode?.parentNode;
+
+            if (!firstDisabledMenuItem) {
+                return;
+            }
+
+            mainMenuItems.forEach(menuItem => {
+                const isDisabled = menuItem.querySelector('a > span.disabled');
```

Comment:
```suggestion
                const isDisabled = menuItem.querySelector('a > span.disabled') !== null;
```

a variable named `isDisabled` makes everyone think it is a boolean, while your current code has the type `Element|null`

Reply:
Correct! Fixed. Thanks!